pkgs/tools/misc/pastebinit/default.nix

```diff
@@ -3,47 +3,74 @@
 , fetchurl
 , fetchpatch
 , python3
+, libxslt
+, docbook-xsl-ns
+, installShellFiles
+, wget
+, cacert
 }:
-stdenv.mkDerivation rec {
-  version = "1.5";
+stdenv.mkDerivation
+rec {
+  version = "1.6.2-1";
```

Comment:
I think the `-1` postfix is part of the ubuntu/debian packaging and doesn't need to be in the Nix package